pkg/cloudprovider/aws/apis/v1alpha1/provider_defaults.go

```diff
@@ -65,3 +68,35 @@ func (c *Constraints) defaultAMIFamily() {
 	}
 	c.AMIFamily = &AMIFamilyAL2
 }
+
+func (c *Constraints) defaultBlockDeviceMappings() {
+	if c.LaunchTemplate != nil {
+		return
+	}
+	if len(c.BlockDeviceMappings) != 0 {
+		return
+	}
+	defaultEBS := BlockDevice{
+		DeleteOnTermination: aws.Bool(true),
+		Encrypted:           aws.Bool(true),
+		VolumeSize:          resource.NewScaledQuantity(20, resource.Giga),
+	}
+	switch aws.StringValue(c.AMIFamily) {
+	case AMIFamilyAL2, AMIFamilyUbuntu:
+		c.BlockDeviceMappings = append(c.BlockDeviceMappings, &BlockDeviceMapping{
+			DeviceName: aws.String("/dev/xvda"),
+			EBS:        &defaultEBS,
+		})
+	case AMIFamilyBottlerocket:
+		c.BlockDeviceMappings = append(c.BlockDeviceMappings, &BlockDeviceMapping{
+			DeviceName: aws.String("/dev/xvdb"),
+			EBS:        &defaultEBS,
+		})
+		defaultEBS.VolumeSize = resource.NewScaledQuantity(4, resource.Giga)
```

Comment:
This sets both /dev/xvda and /dev/xvdb to 4GB, doesn't it?

Reply:
And it does, just noticed the failing test :laughing:

Reply:
👍  you are 100% correct :)